### DISCUSSION TITLE
Changes wont Apply?

### DISCUSSION THREAD

**Discussion by MConsuela** [mastodon/mastodon#28067] [Q&A]
Hello, i did some changes according to these Tutorials

https://draklyckan.se/2021/11/how-to-increase-the-character-limit-for-toots-in-mastodon/
https://stefanbohacek.com/blog/how-to-increase-the-number-of-mastodon-poll-options/

now i did 

> cd live
> RAILS_ENV=production bundle exec rails assets:precompile
> exit
> systemctl restart mastodon-sidekiq
> systemctl reload mastodon-web
> systemctl restart mastodon-streaming

and when i visit website.com/api/v2/instance  it tells me about the Changes, but they arent actually applied and im stuck with both old limitations?

also second question is there a tutorial to Increase the Limit of the "Extra Fields" from 4 to X in the User Profile like in Glitch-Soc?

**Comment by StrangeMinds1960** [comment]
I've just changed the character limit on my instance with the instructions you linked to with one caveat.  When carrying out the first instruction, I found compose_form.js didn't exist, but compose_form.jsx did, and seemed to match what I needed to edit.

Increased the character limit to 1,000 and have succeeded in posting a toot of 936 characters.

Are you seeing any errors after your changes?

**Reply by MConsuela** [reply]
yep the JSX is the one i edited, it even turns up in my API as 10000 but the Browser still says 500.

**Reply by StrangeMinds1960** [reply]
My browser showed 1,000

I don't think this is the answer (because I know of one instance that claims a limit of 65,000 characters), but have you tried re-doing it with a limit of 1,000 characters in case there's some hard-coded limit elsewhere?

Also (and I haven't looked into this), would the poll settings you changed have affected the character limit in any way?

**Reply by StrangeMinds1960** [reply]
Finally, you did refresh you browser tab after making the changes?

**Reply by MConsuela** [reply]
Thanks, somehow it just worked, before it always said "its already newest version" now it actually did compile.   

suppose u dont know if theres a way to st extra fields maximum? :)

**Reply by StrangeMinds1960** [reply]
Ah, so it didn't compile at first?

Don't know about the extra fields

**Reply by MConsuela** [reply]
yes Thanks, i dont know why it did now but it works perfectly.

![3RDtxSQ](https://github.com/mastodon/mastodon/assets/151952217/02cf644b-87fa-462a-b1fb-bd942e40f31b)
